packages/office-ui-fabric-react/src/components/Button/DefaultButton/DefaultButton.tsx

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-unused-variable no-unused-imports */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable no-unused-imports*/
+import {
+  BaseComponent,
+  getNativeProps,
+  anchorProperties,
+  buttonProperties
+} from '../../../Utilities';
+
+import { ButtonBase } from '../ButtonBase/ButtonBase';
+import { IButtonProps } from '../ButtonBase/ButtonBase.Props';
+import './DefaultButton.scss';
+
+export class DefaultButton extends BaseComponent<IButtonProps, any> {
```

Comment:
Update: Micah is now looking at keeping the Button component, and deprecating the ButtonType prop. If ButtonType is provided it would render the desired button (e.g. PrimaryButton) and if it's not, it would render the simple `<button>` element. By doing this we can deprecate the existing functionality without any breaking changes, and not have to call it ButtonBase when we prefer Button.